schemas/org.w3/PerformanceNavigationTiming/jsonschema/1-0-0

```diff
@@ -0,0 +1,231 @@
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp immediately after the browser receives the last byte of the resource or immediately before the transport connection is closed in milliseconds, whichever comes first."
+        },
+        "unloadEventStart": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately after the current document's unload event handler starts in milliseconds."
+        },
+        "unloadEventEnd": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately after the current document's unload event handler completes in milliseconds."
+        },
+        "domInteractive": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately before the user agent sets the document's readyState to 'interactive' in milliseconds."
+        },
+        "domContentLoadedEventStart": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately before the current document's DOMContentLoaded event handler starts in milliseconds."
+        },
+        "domContentLoadedEventEnd": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately after the current document's DOMContentLoaded event handler completes in milliseconds."
+        },
+        "domComplete": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately before the user agent sets the document's readyState to 'complete' in milliseconds."
+        },
+        "loadEventStart": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately after the current document's load event handler starts in milliseconds."
+        },
+        "loadEventEnd": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp representing the time immediately after the current document's load event handler completes in milliseconds."
+        },
+        "startTime": {
+            "type": [ "integer", "null" ],
+            "maximum": 0,
+            "minimum": 0,
+            "description": "Returns a DOMHighResTimeStamp with a value of 0."
```

Comment:
Just noticed this property – is it actually useful if it can only have the value 0?

Reply:
Can't find this reference unfortunately, only the discussions as putting this together with the timing of `unload` timestamps but this has been rejected.
Will remove.